service/github/check.go

```diff
@@ -78,9 +78,15 @@ func (ch *Check) GetResult() *CheckResult {
 	return ch.result
 }
 
+func (ch *Check) SetTool(toolName string, level string) {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method Check.SetTool should have comment or be unexported
<!-- __reviewdog__:ChAzNDU2N2FkNjVmNDBmN2IzEg9nb2xpbnQtY2lyY2xlY2k= -->